### PR TITLE
feat: add remote doc examples

### PR DESCRIPTION
**Pull request by ThibaultFy**
# Description

Add the files to make the examples runable by `run-ci.py`.
✅ ci: https://github.com/owkin/substra-ci/actions/runs/6199350859/job/16831707968

closes FL-994

Need PR on three orgs skaffold profile (cf https://linear.app/owkin/issue/FL-1208/add-skaffold-profile-to-launch-three-organizations) to be merged before

# Companion 

- https://github.com/Substra/substra-documentation/pull/357
- https://github.com/owkin/substra-ci/pull/121

### CODE REVIEW

File: examples/substra_core/diabetes_example/run_diabetes.py [hunk 1]
```diff
@@ -380,7 +401,8 @@
 }
 
 for client_id, key in local_task_1_keys.items():
-    print(f"Status of task {key} on client {client_id}: {clients[client_id].get_task(key).status}")
+    wait_task(client=clients[client_id], key=key)
```
Comment (SdgJlbl) [inline examples/substra_core/diabetes_example/run_diabetes.py:404]: do we need to wait after each task?

**Review by SdgJlbl — APPROVED** [review]
Thanks

File: examples/substra_core/diabetes_example/run_diabetes.py (hunk 1 above)
Comment (ThibaultFy) [reply examples/substra_core/diabetes_example/run_diabetes.py:404]: There is a need to wait for the synchronisation with the orchestrator between each task (cf `wait_for_asset_synchronized` in substra-tests that does that).
As I didn't wanted to write some complicated code, I ended up with this small "wait task" function...

Comment (SdgJlbl) [reply examples/substra_core/diabetes_example/run_diabetes.py:404]: ok, thanks for the explanation

**Pull request merged**